### PR TITLE
Create polling component

### PR DESCRIPTION
**Pull request by garrensmith**
Update active tasks to use the polling component

### CODE REVIEW

File: app/addons/activetasks/assets/less/activetasks.less [hunk 1]
```diff
@@ -188,54 +188,47 @@
...
+//   .polling-interval-time-label {
+//     display: inline-block;
+//     float: right;
+//     margin-right: 0;
+//     cursor: default;
+//   }
+//   #polling-range {
+//     width: 225px;
+//   }
+// }
```
Comment (millayr) [inline app/addons/activetasks/assets/less/activetasks.less]: I'd recommend removing these lines if they're going to be commented out.

File: app/addons/components/components/polling.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  if (secs === 0 || secs > max || secs < min) {
+    return 'off';
+  }
+
+  if (secs < 60 && units === 'minute') {
+    return `${secs} seconds`;
+  }
+
+  let displayValue = secs;
+  if (units === "minute") {
```
Comment (millayr) [inline app/addons/components/components/polling.js]: minor but stay consistent with single or double quotes on string comparisons

**Comment by millayr** [comment]
+1 after my minor comments are discussed/addressed

**Comment by garrensmith** [comment]
Passed over here https://travis-ci.org/garrensmith/couchdb-fauxton/builds/174136630

**Comment by garrensmith** [comment]
Merged

**Pull request closed without merging**